lib/Zonemaster/Backend/Validator.pm

```diff
@@ -96,6 +98,9 @@ Readonly my $RELAXED_DOMAIN_NAME_RE => qr/^[.]$|^.{2,254}$/;
 Readonly my $TEST_ID_RE             => qr/^[0-9a-f]{16}$/;
 Readonly my $USERNAME_RE            => qr/^[a-z0-9-.@]{1,50}$/i;
 
+# Boolean 0 or 1
+Readonly my $BOOL_RE            => qr/^(0|1)$/;
```

Comment:
What do you think about making the values `true` and `false`? That way the configuration file itself is more readable. (I believe the best way to implement it would be to have untaint_bool perform the conversion to 1 and 0.)

Reply:
I have extended the accepted values for boolean to support `true`, `false`, `yes`, `no`, `1` and `0`.

Reply:
Could you update the configuration documentation too?